test/integration/basic_test.lua

```diff
@@ -27,17 +27,17 @@ g.cluster = cartridge_helpers.Cluster:new({
             uuid = cartridge_helpers.uuid('b'),
             roles = { 'vshard-storage' },
             servers = {
-                { instance_uuid = cartridge_helpers.uuid('b', 1), },
-                { instance_uuid = cartridge_helpers.uuid('b', 2), },
+                { instance_uuid = cartridge_helpers.uuid('b', 1), env = {TARANTOOL_HTTP_ENABLED = 'false'} },
+                { instance_uuid = cartridge_helpers.uuid('b', 2), env = {TARANTOOL_HTTP_ENABLED = 'false'} },
             },
         },
         {
             alias = 'storage-2',
             uuid = cartridge_helpers.uuid('c'),
             roles = { 'vshard-storage' },
             servers = {
-                { instance_uuid = cartridge_helpers.uuid('c', 1), },
-                { instance_uuid = cartridge_helpers.uuid('c', 2), },
+                { instance_uuid = cartridge_helpers.uuid('c', 1), env = {TARANTOOL_HTTP_ENABLED = 'false'} },
+                { instance_uuid = cartridge_helpers.uuid('c', 2), env = {TARANTOOL_HTTP_ENABLED = 'false'} },
```

Comment:
I'd suggest adding an assertion somewhere in the test:

```lua
    t.assert_equals(
        g.cluster:server('storage-2-2'):http_request('get', '/', {raise = false}),
        {status = 595, reason = "Couldn't connect to server"}
    )
```

It'll prove that the env actually works as expected.

Reply:
Nica catch, added